src/OpenTelemetry.AutoInstrumentation/Configurations/LogSettings.cs

```diff
@@ -28,20 +28,20 @@ internal class LogSettings : Settings
     public bool IncludeFormattedMessage { get; private set; }
 
     /// <summary>
-    /// Gets a value indicating whether the console exporter is enabled.
+    /// Gets the list of enabled instrumentations.
     /// </summary>
-    public bool ConsoleExporterEnabled { get; private set; }
+    public IReadOnlyList<LogInstrumentation> EnabledInstrumentations { get; private set; } = new List<LogInstrumentation>();
 
     /// <summary>
-    /// Gets the list of enabled instrumentations.
+    /// Gets or sets a value indicating whether the console exporter is enabled.
     /// </summary>
-    public IReadOnlyList<LogInstrumentation> EnabledInstrumentations { get; private set; } = new List<LogInstrumentation>();
+    private bool ConsoleExporterEnabled { get; set; }
```

Comment:
Why do have a private property? Should it not be just a field?
However, if we add `static` back to the `Parse` method I think we would not need even a field.

Same comment applies to other places.